alws/utils/pulp_client.py

```diff
@@ -203,6 +205,42 @@ async def upload_comps(self, data: dict) -> typing.List[str]:
         task_result = await self.wait_for_task(task['task'])
         return task_result['created_resources']
 
+    async def upload_local_file(self, file_path: str, sha256: str,
+                                chunk_size: int = 52428800):
+        file_size = os.path.getsize(file_path)
+        chunks = math.ceil(file_size / chunk_size)
+        start = 0
+        upload_href = (await self.request(
+            'POST', 'pulp/api/v3/uploads/', json={'size': file_size}
+        ))['pulp_href']
```

Comment:
can we reuse already existent endpoints for `uploads` from `pulp_client`?

Reply:
I tried to do as less changes to existing functionality as possible, didn't want to break the `comps` upload. Can rework both of them to use the same code